st2actions/st2actions/bootstrap/runnersregistrar.py

```diff
@@ -436,6 +436,31 @@
             }
         },
         'runner_module': 'st2actions.runners.windows_script_runner'
+    },
+    {
+        'name': 'cloudslang',
+        'aliases': [],
+        'description': 'A runner to execute cloudslang flows.',
+        'enabled': True,
+        'runner_parameters': {
+            'flow_path': {
+                'description': 'Path to a flow to run',
+                'type': 'string',
+                'required': True
+            },
+            'inputs': {
```

Comment:
I see you split and convert this string into dict inside the runner code.

To make this more robust and reflect the actual type, you should specify `'type': 'object'` here. This way user can directly specify an object (dict).

When you interact with the CLI you can still specify parameter as a string in `input1=val1,input2=val2` format because the CLI automatically converts it into dict for arguments with type `object`. In addition to the string format, you can also specify objects using JSON notation (e.g. `inputs='{"input1": "val1", "input2": "val2"}'`)